Source/AliveLibAO/Abe.cpp

```diff
@@ -2258,6 +2260,228 @@ __int16 Abe::TryMountElum_42E600()
     return eAbeStates::State_0_Idle_423520;
 }
 
+void Abe::BulletDamage_4220B0(Bullet* pBullet)
+{
+    PSX_RECT rect = {};
+    VGetBoundingRect_418120(&rect, 1);
+
+    enum class ShootKind
+    {
+        eEverythingElse_0 = 0,
+        eHanging_1 = 1,
+        eRolling_2 = 2,
+    };
+
+    if (field_FC_current_motion != eAbeStates::State_137_ElumUnmountBegin_42E2B0
```

Comment:
wouldn't it be better to invert this + return early so the rest of the func isn't so indented?

Reply:
AE doesn't have it, do it for AE also?